### DISCUSSION TITLE
Wix Msi linking with 3.11 vs 4.0 "reusecab" option gone or renamed?

### DISCUSSION THREAD

**Discussion by woscostametaxa** [wixtoolset/issues#7361] [Questions] [answered]
Light in Wix 3.11.2 had the "-reusecab" argument which in combination with the cabcache argument helped speed up Msi generation for my usecase by quite a bit - I need Msi packages (more than 10) to have 3 languages, so I would create one Msi per lang and then embed the transformations.

When comparing my current build with wix 3.11.2 the runtime of my automation is about 13 minutes, and the rc4 (with wix build) runtime is 18 minutes. When trying `wix build --help` I don't see the possibility to add the reusecab any more - has this been removed, or automatically added when using things like cabcache and the build is just slower due to changes in architecture?

**Comment by robmen** [comment, marked as answer]
Specifying a cab cache path should be all that is necessary to activate the feature in WiX v4.
